Docker_files/ocsci_container/Dockerfile_ocsci

```diff
@@ -0,0 +1,47 @@
+FROM registry.redhat.io/ubi8/ubi
+
+# Based on https://catalog.redhat.com/software/containers/ubi8/python-38/5dde9cacbed8bd164a0af24a
+
+# Use this dockerfile to create ocs-ci infra and run test based on argument
+
+# Configure k8s repository to install kubectl packages.
+COPY k8s.repo /etc/yum.repos.d/k8s.repo
+
+# Python package installation.
+RUN INSTALL_PKGS="python38 python38-devel python38-setuptools python38-pip \
+      libffi-devel libcurl-devel openssl-devel libxslt-devel libxml2-devel libtool-ltdl enchant glibc-langpack-en redhat-rpm-config \
+      git gcc kubectl" && \
+    yum -y module enable python38:3.8 && \
+    yum -y --setopt=tsflags=nodocs install $INSTALL_PKGS && \
+    rpm -V $INSTALL_PKGS && \
+    yum -y clean all --enablerepo='*' && \
+    rm -rf /var/cache/yum
+
+# Environment variables containing various file locations required by install-ocs-ci.sh script
+ENV OCSCI_INSTALL_DIR=/opt/ocs-ci \
+    CLUSTER_DIR=/opt/cluster
```

Comment:
I guess this CLUSTER_DIR is not used anywhere

Reply:
done